src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysis.TaintedDataOperationVisitor.cs

```diff
@@ -415,6 +424,24 @@ private bool IsSanitizingMethod(IMethodSymbol method)
                 return false;
             }
 
+            /// <summary>
+            /// Determines if untaint the instance after calling the method.
```

Comment:
>Determines if untaint the instance after calling the method [](start = 16, length = 59)

Determines if the method untaints its parameters (because we assume it's being call from if statement predicate or something).